consensus/src/liveness/round_state.rs

```diff
@@ -332,7 +352,8 @@ impl RoundState {
         let timeout = self
             .time_interval
             .get_round_duration(round_index_after_committed_round)
-            * multiplier;
+            * multiplier
+            + backoff_timeout;
```

Comment:
we probably unify how the timeout is calculated, currently there's exponential backoff by current round - latest committed round